### PR TITLE
README: fix reference to bb docs, add note on templates

### PR DESCRIPTION
**Pull request by orangecms**
Signed-off-by: Daniel Maslowski <info@orangecms.org>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -484,7 +491,11 @@ assuming your kernel is configured to work that way.
 u-root can create an initramfs in two different modes, specified by `-build`:
 
 *   `bb`/`gbb` mode: One busybox-like binary comprising all the Go tools you ask to
-    include. See [here for how it works](pkg/bb/README.md).
+    include, which can be [templates](#templates) and/or individual commands.
```
Comment (hugelgupf) [inline README.md:494]: makes it sound like templates are only possible with gbb/bb mode -- they work with any mode

File: README.md [hunk 2]
```diff
@@ -484,7 +491,11 @@ assuming your kernel is configured to work that way.
...
+    See [the gobusybox README for how it works](https://github.com/u-root/gobusybox).
+    **NOTE**: `gbb` is now the default variant. `bb` is an older implementation
```
Comment (hugelgupf) [inline README.md:496]: How about we just stop referring to bb from now on?

Comment (orangecms) [reply README.md:496]: yea, good point, I'll rewrite it and drop it

File: README.md (hunk 1 above)
Comment (orangecms) [reply README.md:494]: gotcha - I'll drop that

File: README.md [hunk 3]
```diff
@@ -52,9 +52,16 @@ go install github.com/u-root/u-root
...
 need to add `$GOPATH/bin` to your `$PATH`.**
 
+## Templates
+
+To quickly specify a set of commands from u-root, you can use any of the
+templates as defined in [templates.go](templates.go).
+
+## Examples
+
 You can now use the u-root command to build an initramfs. Here are some examples
```
Comment (rminnich) [inline README.md:62]: let's get rid of the word 'now', it made sense about 9 years ago but :-)

**Review by rminnich — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (orangecms) [reply README.md:62]: Yea, I'll skim through the entire README again and see what else could be reworded - the next part here is a lengthy list of examples and loose explanations. I see some potential for reordering to give them some more structure. More PRs to come!

**Pull request merged**